x-pack/plugins/ml/server/lib/ml_client/search.ts

```diff
@@ -34,11 +34,15 @@ export function searchProvider(
   ): Promise<TransportResult<estypes.SearchResponse<T>, unknown>> {
     await jobIdsCheck('anomaly-detector', jobIds);
     const { asInternalUser } = client;
-    const resp = await asInternalUser.search<T>({
-      ...searchParams,
-      index: ML_RESULTS_INDEX_PATTERN,
-    });
+    const resp = await asInternalUser.search<T>(
+      {
+        ...searchParams,
+        index: ML_RESULTS_INDEX_PATTERN,
+      },
+      { meta: true }
```

Comment:
`anomalySearch` is a simple wrapper around a standard es `search` call.
For consistency with how other `search` calls are made I think this should also not return a `body`